dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -826,10 +873,11 @@ public void constantField_notUpdatedCache_whenChanged() throws Exception{
      *
      */
     @Test
-    public void containerArchived_PageShouldResolve() throws Exception {
+    @UseDataProvider("cases")
+    public void containerArchived_PageShouldResolve(final Container testCaseContainer, final Template template) throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5025402)